### DISCUSSION TITLE
Does the thermocouple need to be calibrated?

### DISCUSSION THREAD

**Discussion by ambarusa** [Zer0-bit/gaggiuino#282] [General]
I ran a few tests after my sensor arrived, and I saw that it measures a bit higher temperature. Did you have similar issues? If so, do you know if it's linear, so an offset would do the work?

**Comment by Zer0-bit** [comment]
No need for calibration. there is an offset in the settings that can account for this if the need be, you can do a simple test once at least base install is ready to find out if it's needed or not but to my knowledge until now nobody had the need to calibrate theirs.

**Comment by ambarusa** [comment]
Ok, I see now in your code. So it may happen, that my thermocouple is not accurate. So far I can see around 4 C difference in the room temperature compared to other sensors.

**Reply by banoz** [reply]
You might have swapped TC poles. Try reverse the polarity.

**Comment by ambarusa** [comment]
I made a few tests, my assumption is that it also depends a lot on the input source. I set in the fridge with a big power supply (instead of my laptop usb port), and a few more sensors, and all were approximately the same. Thanks guys
